apps/src/sites/studio/pages/levels/editors/fields/_skill_evaluations.js

```diff
@@ -6,13 +6,21 @@ import SkillEvaluationSettings from '@cdo/apps/levelbuilder/skills/SkillEvaluati
 import getScriptData from '@cdo/apps/util/getScriptData';
 
 $(document).ready(function () {
+  const aiPromptModificationInput = $(
+    'input#level_additional_ai_evaluation_instructions'
+  );
```

Comment:
ok, I think I see what's happening. You hide the actual input field and then manage it's value with the state from the React component. On face value this feels a little convoluted to me. Is it a pattern we use for other levelbuilder fields?

Reply:
Yeah, this was tricky!  [We use it in ImageInput as well... ](https://github.com/code-dot-org/code-dot-org/blob/a9e22fbf83dcd1f78f37be637a6803e6ef779dee/apps/src/sites/studio/pages/levels/editors/fields/_bubble_choice_sublevel.js#L8).  I didn't love this solution but found it hard to find examples of React components in levelbuilder level pages...

Reply:
@sanchitmalhotra126 I have some fuzzy recollection that you made React component rendering in levelbuilder UI easier? If not, ignore me 😁 Just seeing if there's a "standard" "good" way of doing this. 

@kobryan0619 Not blocking, now just asking for my own long-term edification.